src/content/blog/2023-01-10-eslint-2022-year-review.md

```diff
@@ -0,0 +1,112 @@
+
+## Income review
+
+During 2022, ESLint received $197,267.73 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+
+| **Source** | **Amount** |
+|-----------|-------------|
+| Open Collective | $127,741.96 |
+| GitHub Sponsors | $26,606.61 |
+| Tidelift | $29,088.00 |
+| Carbon Ads | $13,831.16 |
+| Stackaid.us | $77.54 |
+| **Total** | **$197,267.73** |
+
+As usual, most of our donations come through Open Collective followed by GitHub Sponsors. Our Tidelift agreement also provided a good amount of income as did the ad on our website.
+
+We are extremely grateful for the generous donations from companies and individuals that allow the project to continue development.
+
+## Expenses review
+
+In 2022, we spent $133,536.57 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. See the table below for an overview.
+
+| **Category** | **Amount** |
+|-----------|-------------|
+| Maintenance and Development | $97,516.23 |
+| Contributor Pool | $7,900.00 |
+| Guest Blog Posts | $600.00 |
+| Community Logo Design | $2,700.00 |
+| Documentation Project | $4,147.50 |
+| Tech Writer Application Time | $1,500.00 |
+| Open Source Collective Fee | $12,172.84 |
+| Community Management | $7,000.00 |
+| **Total** | **$133,536.57** |
+
+As a reminder, the Contributor Pool is money we award for contributions to people who are not on the ESLint team as a way to thank them for making impactful contributions to the project. Community Management refers to the ongoing work required to manage the ESLint Community.
+
+## Supporting our dependencies and ecosystem
+
+In 2022, the ESLint project donated $14,350.00 USD to ecosystem projects and our dependencies. This is part of our ongoing commitment to supporting projects that are important to ESLint.
+
+| **Project** | **Donations** |
+|-----------|-------------|
+| lint-staged | $2,450.00 |
+| Ajv | $1,700.00 |
+| debug | $1,700.00 |
+| eslint-plugin-import | $1,700.00 |
+| jsx-eslint | $1,700.00 |
+| Sindre Sorhus | $1,700.00 |
+| typescript-eslint | $1,700.00 |
+| Rollup | $900.00 |
+| Eleventy | $800.00 |
+| **Total** | **$14,350.00** |
+
+## Looking forward to 2023
+
+While ESLint had a fantastic 2022, we are looking forward to an even better 2023. With the ESLint Community and documentation projects running, we expect to make the ESLint ecosystem even stronger. We are also planning to start on a [complete rewrite](https://github.com/eslint/eslint/discussions/16557) from the ground up to make sure that ESLint is ready for the next ten years of JavaScript development. This process will likely take place in small, incremental steps as we think through how JavaScript development has changed and will continue to evolve in the future.
```

Comment:
```suggestion
While ESLint had a fantastic 2022, we are looking forward to an even better 2023. With the ESLint Community and documentation projects running, we expect to make the ESLint ecosystem even stronger. We are also planning to start a [complete rewrite](https://github.com/eslint/eslint/discussions/16557) from the ground to make sure that ESLint is ready for the next ten years of JavaScript development. This process will likely take place in small, incremental steps as we think through how JavaScript development has changed and will continue to evolve in the future.
```
maybe?

Reply:
No. The phrase is “ground up” so you can’t remove the second word.

Reply:
Can we change `We are also planning to start on a` to `We are also planning to start a`? Or is it fine as well?